internal/server/iigointernal/orchestration.go

```diff
@@ -70,6 +70,10 @@ func RunIIGO(g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.C
 		judgeID:           g.JudgeID,
 		internalIIGOCache: []shared.Accountability{},
 	}
+	executiveBranch.monitoring = &monitoring
+	legislativeBranch.monitoring = &monitoring
+	judicialBranch.monitoring = &monitoring
```

Comment:
do you mutate these in the branches

Reply:
why pointers?

Reply:
I'm passing in references to the same monitoring object so that they can populate the internal iigo cache variable in monitoring. Is this correct?

Reply:
alright, a bit dangerous but ok ._.